docs/tac-skills.md

```diff
@@ -40,7 +42,7 @@ You must set the start time to a point in the future `start_time: Nov 10 2019  1
 
 ### Run the TAC controller AEA
 ``` bash
-aea run
+aea run --connections oef
```

Comment:
an alternative, completely optional: we can use `aea config set agent.default_connection oef` at some point earlier in the tutorial. The same for the other skills.

Reply:
Yeah we could do that in another iteration. Will leave it for now.